### PR TITLE
Story4

### PR DESCRIPTION
**Pull request by robin-hodge**
As a: player
I want: a timer counting down and results at the end
So that: I can try my best to type within the time limit

Requirements:

Timer starts at 60 seconds

It only starts counting down once the user has started typing

When the timer ends, stop the game from accepting input and display the results of the game as per the wireframe

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -68,12 +70,49 @@ textInput.addEventListener('keyup', event => {
                 top: scrollPixels += lineHeight,
                 behavior: 'smooth'
             })
-        
+
         }
         upcomingWord.style.color = upcomingWordColour
 
+
```
Comment (mporam) [inline script.js]: unnecessary whitespace

File: script.js [hunk 2]
```diff
@@ -68,12 +70,49 @@ textInput.addEventListener('keyup', event => {
...
+        const wordsPerMinuteResult = document.querySelector('#wordsPerMinuteResult')
+        const accuracyResult = document.querySelector('#accuracyResult')
+
+        let totalAttemptedWords = wordIndexCount + 1
+        let wordsPerMinute = numberOfValidWords
+        let accuracy = Math.round((wordsPerMinute / totalAttemptedWords) * 100) + '%'
+        wordsPerMinuteResult.innerHTML = wordsPerMinute
+        accuracyResult.innerHTML = accuracy
```
Comment (mporam) [inline script.js]: thinking about this, why are you doing it on every word? this only needs to be calculated when the time is up at the end

**Review by mporam — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (melody-sylvestre) [reply script.js]: Done

File: script.js (hunk 2 above)
Comment (melody-sylvestre) [reply script.js]: Done: we calculate the accuracy and print the results in the HTML page only when the time is up

File: script.js [hunk 3]
```diff
@@ -68,12 +70,46 @@ textInput.addEventListener('keyup', event => {
...
+
+textInput.addEventListener('keyup', () => {
+    const countdownForDisplay = setInterval(() => {
+        timer.innerHTML = countdown
+        countdown--
+    }, 1000)
+    const countdownForInput = setTimeout(() => {
+        document.querySelector("#resultsPopup").style.display = 'block'
+        document.getElementById("textToTypeContainer").scroll({
+            top: scrollPixels = 0,
```
Comment (mporam) [inline script.js]: this is dangerous, please fix

**Review by mporam — CHANGES_REQUESTED** [review]

File: script.js (hunk 3 above)
Comment (melody-sylvestre) [reply script.js]: Done

**Review by mporam — APPROVED** [review]
